modules/pouchdb/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@nuxtjs/pouchdb",
+  "version": "0.1.0",
+  "description": "Adds reactive PouchDB bindings to Nuxt",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nuxt/modules.git"
+  },
+  "homepage": "https://github.com/nuxt/modules/modules/pouchdb",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "private": false,
+  "author": "Stefan Buhrmester <buhrmi@gmail.com>",
+  "license": "MIT",
+  "dependencies": {
+    "pouchdb": "*",
```

Comment:
Would you please prefer caret version dependencies (`^x.y.x`)? `*` can break dependent packages on pouchdb's major bumps.